style/base.css

```diff
@@ -134,6 +134,11 @@
   border-top-right-radius: 0;
 }
 
+.jp-NotebookPanel-toolbar.je-ViewOnlyNotebookToolbar {
+  border: 3px solid var(--je-slate-blue) !important;
```

Comment:
Good catch, I missed that it has a border in view-only mode on the mockup (and is taller).

We can include this here, for increasing the height we could use min-height but I think it's best to make it a follow-up issue as interacts with the shadow for dialogs in a way which is hard to solve generically.

Reply:
Yes, I think so too. I had a bunch of problems with making it taller, so it's better handled separately.